### ISSUE TITLE
Generate Flow Diagram functionality should be expanded. 

### ISSUE THREAD

**Issue by richiesun** [abynim/UserFlows.sketchplugin#84]
I use User Flows during my design phase to help me organize my screens/flow as I think of them. It would be helpful to "Generate Flow Diagram", have the plugin move my affected flow to another page where I can further create links / remove links / move artboards. The plugin currently flattens everything. 

Seems the current logical workflow is to:
1. Design my screens in my normal orientation on my main sketch page. 
2. Dupe the artboards for a flow into a new page, then rearrange and space them so they look like a conventional flow chart. 
3. Then create links on the duped page to show the flow. 
4. Export to team for approval. 
5. Make edits to original screens. 
6. Dupe artboards again and place in flows page. 
7. Redo links. 
8. Export for review again. 

Basically, User Flows needs its own space in Sketch where a designer can easily work it into their workflow. The arrangement of artboards to show a flow is different from the arrangement a designer might use when creating all of their screens. Kind of like how Symbols are now on a separate working page. It doesn't clog up the main working page.
